### ISSUE TITLE
Filter for Readrops

### ISSUE THREAD

**Issue by m4rsh** [readrops/Readrops#240]
Problem: Having multiple Readrops sources, you can combine them under a category (as folder). It is only possible to show them all, or only one at a time. I would like to select, which Readrops to show in the results list.

Solution: Add checkboxes for every source as a filter mechanism.

**Comment by Shinokuni** [comment]
What do you call Readrops sources? Feeds? 

I see what you would like to have, but I think that with new/favorites/feed/folder/read+unread filters, there are enough of them. Add another type of filter which would be a kind of temporary folder isn't very relevant to me.

**Comment by m4rsh** [comment]
Yes, I mean feeds. In the end, it's no problem to put every feed in one folder. But then the whole folder structure makes no sense, without the possibility to filter different feeds to combine or separate them. But, that's up to the use case.

**Comment by m4rsh** [comment]
> Couldn't you use multiple folders? 
Yes, I use multiple folders, like: Technology, Science, News....
And under these I have multiple sources/ feeds which relate to the topic. But I can only show them all within the themed folders, or only a single source. But there is no inbetween, which could be easily met by checkboxes. These would be shown in front of each source feed.

>Are you feeds related to only one topic?
Yes, the topic name is the theme/ folder name I mentioned before. 

 >Also I forgot to tell, but I plan to add support for tags, so it should meet your use case.
This could fulfill the function of themes/ topics. Therefore, folders wouldn't be strictly necessary.
